backtester/backtester.cpp

```diff
@@ -226,15 +251,55 @@ void runParamCombos(int i, vector<string>& rangeNames, vector<vector<double>>& r
 	}
 }
 
+void simulateDay(double& currentCash, unordered_map<string, double>& params, size_t& currentTime, double& currentQuote, Action& action,
+	unique_ptr<Algorithm>& algo, Day& day, bool& activePositions, Result& result, Trade& trade, size_t& stockUnits) {
+	for (size_t i = 0; i < day.quotes.size(); ++i) {
+		if (currentCash < params["minimumCash"]) {
```

Comment:
Yeah breaking right after the last sell action that makes your cash drops below the threshold could be good.